### ISSUE TITLE
USWDS - Radio and checkbox: Revert tile border colors

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#5490]
## Summary
Release 3.5.0 inadvertently changed the border color for the Radio and Checkbox tile outlines. We should revert these styles back to the 3.4.0 styles. 

## Details

The tile border colors are defined with disabled color tokens. These settings received an update in 3.5.0, which made the border color darker. Note the color shifts between [3.4.0](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/release-3.4.0/?path=/story/components-form-inputs-radio--radio-tile) and [3.5.0](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/release-3.5.0/?path=/story/components-form-inputs-radio--radio-tile). This could play a role in the visual noise. 

![image](https://github.com/uswds/uswds/assets/93996430/353e03f1-6c4b-47ed-b72a-2c316364828b)

This is the first part of a bigger discussion related to component styles and visual processing disorder in:  https://github.com/uswds/uswds/issues/5487

### To-do
- [ ] Investigate if we should be defining this color with disabled settings
